### ISSUE TITLE
Serverless

### ISSUE THREAD

**Issue by ClashTheBunny** [khuedoan/homelab#86]
It would be nice to have something serverless, like [knative.dev](https://knative.dev) , or one of the other CNCF options: https://landscape.cncf.io/card-mode?category=serverless&license=open-source&grouping=category&sort=stars

**Comment by khuedoan** [comment]
Yep this is on the roadmap https://homelab.khuedoan.com/reference/roadmap/#unplanned

**Comment by khuedoan** [comment]
I abandoned the Knative idea because for most (if not all) of my personal projects, they are better served with traditional HPA + Deployment and the like, because most of them are written in Rust/Go, which only use a tiny amount of resources, so the value of scale to zero is minimal.
